### PR TITLE
br: add a waring about multi br importing

### PR DESCRIPTION
**Pull request by YuJuncen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Added a warning about running multiple BRs in one cluster, which is error-prone.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]

**Comment by YuJuncen** [comment]
/home/circleci/pingcap/docs-cn/sql-plan-management.md
   325:9  error    shouldn't use full-width char:'（' in en sentence   ZH417
   325:9  warning  in en sen, left parenthesis should half-width '('   ZH420L
  325:13  error    shouldn't use full-width char:'）' in en sentence   ZH417
  325:13  warning  in en sen, right parenthesis should half-width ')'  ZH420R

Why an unrelated file annoys the CI...

**Comment by TomShawn** [comment]
> ```
> /home/circleci/pingcap/docs-cn/sql-plan-management.md
>    325:9  error    shouldn't use full-width char:'（' in en sentence   ZH417
>    325:9  warning  in en sen, left parenthesis should half-width '('   ZH420L
>   325:13  error    shouldn't use full-width char:'）' in en sentence   ZH417
>   325:13  warning  in en sen, right parenthesis should half-width ')'  ZH420R
> ```
> 
> Why an unrelated file annoys the CI...

There's nothing wrong with this PR. The error reporting does not block PR merge.

File: br/backup-and-restore-faq.md [hunk 1]
```diff
@@ -170,3 +170,11 @@ BR 不会备份统计信息（v4.0.9 除外）。所以在恢复存档后需要
 BR v4.0.9 备份统计信息使 BR 消耗过多内存，为保证备份过程正常，从 v4.0.10 开始默认关闭备份统计信息的功能。
 
 如果不对表执行 `ANALYZE`，TiDB 会因统计信息不准确而选不中最优化的执行计划。如果查询性能不是重点关注项，可以忽略 `ANALYZE`。
+
+# 是否可以同时启动多个 BR 对单个集群进行恢复？
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:174]: 这里的 BR 是否有单位呢？集群，实例，节点，还是其他单位呢？
```suggestion
## 是否可以同时启动多个 BR 对单个集群进行恢复？
```

File: br/backup-and-restore-faq.md [hunk 2]
```diff
@@ -170,3 +170,11 @@ BR 不会备份统计信息（v4.0.9 除外）。所以在恢复存档后需要
...
+
+**强烈不建议**在单个集群中同时使用多个 BR 进行恢复，原因如下：
+
+1. BR 在恢复时，会修改 PD 的一些全局配置。在多个 BR 同时进行恢复的时候，这些配置可能会被错误地覆写，导致集群状态异常。
+2. 同时，因为 BR 在恢复的时候会占用大量集群资源，事实上并行恢复能获得的速度提升也非常有限。
+3. 多个 BR 并行恢复的场景没有经过测试，无法保证成功。
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:180]:
```suggestion
+ BR 在恢复数据时，会修改 PD 的一些全局配置。在多个 BR 同时进行恢复的时候，这些配置可能会被错误地覆写，导致集群状态异常。
+ 因为 BR 在恢复数据的时候会占用大量集群资源，事实上并行恢复能获得的速度提升也非常有限。
+ 多个 BR 并行恢复的场景没有经过测试，无法保证成功。
```

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -261,6 +261,8 @@ br backup raw --pd $PD_ADDR \
 > * 在数据恢复的时候，每个 Peer 分布的位置是随机的，事先并不知道哪个节点将读取哪个文件。
 >
 > 使用共享存储可以避免这些情况。例如，在本地路径上安装 NFS，或使用 S3。利用这些网络存储，各个节点都可以自动读取每个 SST 文件，此时上述注意事项不再适用。
+> 
+> 同时，请注意同一时间对同一个集群只能运行一个恢复任务，否则可能会出现非预期的行为，详见[FAQ](/br/backup-and-restore-faq.md#是否可以同时启动多个-BR-对单个集群进行恢复？)。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:265]:
```suggestion
> 同时，请注意同一时间对同一个集群只能运行一个恢复任务，否则可能会出现非预期的行为，详见 [FAQ](/br/backup-and-restore-faq.md#是否可以同时启动多个-BR-对单个集群进行恢复)。
```

File: br/use-br-command-line-tool.md [hunk 4]
```diff
@@ -261,6 +261,8 @@ br backup raw --pd $PD_ADDR \
 > * 在数据恢复的时候，每个 Peer 分布的位置是随机的，事先并不知道哪个节点将读取哪个文件。
 >
 > 使用共享存储可以避免这些情况。例如，在本地路径上安装 NFS，或使用 S3。利用这些网络存储，各个节点都可以自动读取每个 SST 文件，此时上述注意事项不再适用。
+> 
+> 同时，请注意同一时间对同一个集群只能运行一个恢复任务，否则可能会出现非预期的行为，详见 [FAQ](/br/backup-and-restore-faq.md#是否可以同时启动多个-BR-对单个集群进行恢复)。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:265]:
```suggestion
> 同时，请注意同一时间对同一个集群只能运行一个恢复任务，否则可能会出现非预期的行为，详见 [FAQ](/br/backup-and-restore-faq.md#是否可以同时启动多个-br-对单个集群进行恢复)。
```

Comment (YuJuncen) [reply br/use-br-command-line-tool.md:265]: oops..

File: br/backup-and-restore-faq.md (hunk 1 above)
Comment (TomShawn) [reply br/backup-and-restore-faq.md]: @YuJuncen cc

Comment (YuJuncen) [reply br/backup-and-restore-faq.md]: 换成了“BR 命令”，用来指代 BR CLI。

File: br/backup-and-restore-faq.md [hunk 5]
```diff
@@ -170,3 +170,11 @@ BR 不会备份统计信息（v4.0.9 除外）。所以在恢复存档后需要
 BR v4.0.9 备份统计信息使 BR 消耗过多内存，为保证备份过程正常，从 v4.0.10 开始默认关闭备份统计信息的功能。
 
 如果不对表执行 `ANALYZE`，TiDB 会因统计信息不准确而选不中最优化的执行计划。如果查询性能不是重点关注项，可以忽略 `ANALYZE`。
+
+## 是否可以同时启动多个 BR 命令对单个集群进行恢复？
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:174]:
```suggestion
## 是否可以同时使用多个 BR 命令对单个集群进行恢复？
```

File: br/backup-and-restore-faq.md [hunk 6]
```diff
@@ -170,3 +170,11 @@ BR 不会备份统计信息（v4.0.9 除外）。所以在恢复存档后需要
...
+
+**强烈不建议**在单个集群中同时使用多个 BR 命令进行恢复，原因如下：
+
++ BR 在恢复数据时，会修改 PD 的一些全局配置。在多个 BR 命令同时进行恢复的时候，这些配置可能会被错误地覆写，导致集群状态异常。
```
Comment (TomShawn) [inline br/backup-and-restore-faq.md:178]:
```suggestion
+ BR 在恢复数据时，会修改 PD 的一些全局配置。如果同时使用多个 BR 命令进行恢复，这些配置可能会被错误地覆写，导致集群状态异常。
```

File: br/use-br-command-line-tool.md [hunk 7]
```diff
@@ -261,6 +261,8 @@ br backup raw --pd $PD_ADDR \
 > * 在数据恢复的时候，每个 Peer 分布的位置是随机的，事先并不知道哪个节点将读取哪个文件。
 >
 > 使用共享存储可以避免这些情况。例如，在本地路径上安装 NFS，或使用 S3。利用这些网络存储，各个节点都可以自动读取每个 SST 文件，此时上述注意事项不再适用。
+> 
+> 同时，请注意同一时间对同一个集群只能运行一个恢复任务，否则可能会出现非预期的行为，详见 [FAQ](/br/backup-and-restore-faq.md#是否可以同时启动多个-br-对单个集群进行恢复)。
```
Comment (TomShawn) [inline br/use-br-command-line-tool.md:265]:
```suggestion
> 同时，请注意同一时间对同一个集群只能运行一个恢复任务，否则可能会出现非预期的行为，详见 [FAQ](/br/backup-and-restore-faq.md#是否可以同时使用多个-br-命令对单个集群进行恢复)。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**